### PR TITLE
Misc updates to the getting-started doc

### PR DESCRIPTION
**Pull request by MathewMacDougall**
### Description

Updated some links and misc documentation in `getting-started.md`. I also moved the `.bazelproject` file up one level so that project importing in CLion works better.

What happens currently is that if you import a new bazel project, you need to select "import project view" otherwise the .bazelproject file we have checked in won't be used. Regardless, CLion won't be able to create the project because the `.clwb/.bazelproject` file already exists. CLion looks to generate this folder in the "data path", which is the `src` folder we select initially for the workspace.

Therefore the 2 solutions are to either have people use a different folder for the project data, meaning they'll have to choose a folder other than `src` (and it is not recommended you choose a child folder of the `src` folder), or we have to move the `.bazelproject` file. This is what I chose to do since I think it makes the setup process easier.

Now if you select `src/.bazelproject` as the project view file, everything works fine. A new `src/.clwb/.bazelproject` file is generated but it simply imports our `src/.bazelproject` file so will use our file's contents.

However I'm currently not 100% sure what this would do to people's existing projects, so if anyone wants to test to confirm that'd be great.

### Testing Done

Tested project importing works as expected

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [ ] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

**Review by jonathanlew — COMMENTED** [review]
LGTM. I will test importing tomorrow then approve

**Review by garethellis0 — DISMISSED** [review]
LGTM :+1: :tada:

File: docs/getting-started.md [hunk 1]
```diff
@@ -26,8 +26,8 @@
 
 ## Introduction
 These instructions assume that you have the following accounts setup:
-- Github
-- Slack
+- [Github](https://github.com/login)
+- [Slack](https://slack.com/signin)
```
Comment (henryymliu) [inline docs/getting-started.md]: nit: should this link directly to our `thunderbots.slack.com` domain?

**Review by henryymliu — COMMENTED** [review]
Doc changes LGTM overall, I'll let those that use CLion review the `.bazelproject` changes

**Review by jonathanlew — DISMISSED** [review]
LGTM!

File: docs/getting-started.md (hunk 1 above)
Comment (MathewMacDougall) [reply docs/getting-started.md]: good idea. updated the link

**Review by jonathanlew — APPROVED** [review]
👍

**Review by henryymliu — APPROVED** [review]
🚢🤡

**Review by garethellis0 — APPROVED** [review]
LGTM :+1: :tada:

**Pull request merged**